pkg/iac/terraform/state/enumerator/glob.go

```diff
@@ -15,6 +15,11 @@ func GlobS3(path string) (prefix string, pattern string) {
 	return
 }
 
+func GlobGS(path string) (prefix string, pattern string) {
```

Comment:
it looks like this and a good part of the enumeration method is duplicated from the s3 code. Do you think we could find a way to share the mathcing code ?

Reply:
I was thinking the same to be honest, I'll take a look at it this evening and see if I can offload some code to common functions 👍🏼

Reply:
I've moved some code around, to a `common.go`, removed some wrapper functions that didn't add a lot of value.

Reply:
Changed nothing in the functionality, just renamed functions and removed duplicate code.